crates/bevy_render/Cargo.toml

```diff
@@ -60,9 +60,11 @@ image = { version = "0.24", default-features = false }
 codespan-reporting = "0.11.0"
 # `fragile-send-sync-non-atomic-wasm` feature means we can't use WASM threads for rendering
 # It is enabled for now to avoid having to do a significant overhaul of the renderer just for wasm
-wgpu = { version = "0.17.1", features = ["naga", "fragile-send-sync-non-atomic-wasm"] }
-naga = { version = "0.13.0", features = ["wgsl-in"] }
-naga_oil = "0.10"
+wgpu = { version = "0.18", features = ["naga", "fragile-send-sync-non-atomic-wasm"] }
+naga = { version = "0.14.0", features = ["wgsl-in"] }
+# naga_oil = "0.10"
+naga_oil = { git = "https://github.com/Elabajaba/naga_oil.git", branch = "naga014" }
+# naga_oil = { path = "D:/programming/naga_oil" }
```

Comment:
Of course this will need to be changed once naga_oil is fixed and released, but noting it to reduce cognitive load.